carAPI/src/test/java/boundary/CarResourceTests.java

```diff
@@ -0,0 +1,88 @@
+package boundary;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.everis.boundary.CarResource;
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CarResourceTests {
+
+	@InjectMocks
+	private CarResource carResource;
+	
+	@Mock
+	private CarService carService;
+	
+	@Mock
+	private UriInfo uriInfo;
+	
+	private Car car;
+	
+	@Before
+	public void setUp() throws Exception{
+		car = new Car();
+		car.setId(3);
+		car.setBrand("TestCar");
+		car.setCountry("Spain");
+		car.setRegistration(LocalDateTime.now());
+		car.setCreatedAt(LocalDateTime.now());
+		car.setLastUpdated(LocalDateTime.now());
+	}
+	
+	@Test
+	public void testGetCars() {
```

Comment:
Tests are looking good, also keep in mind:
Unit tests goal is to test the most significant use cases that a single method can handle. That means, try to explorer also the weird use cases in the tests. For instance, what if the CarService instead of returning a proper list of cars, returns a null. Having unit tests for corner cases is a good idea from my point of view. Always try to tests unexpected behaviours!